teps/0080-support-domainscoped-parameterresult-names.md

```diff
@@ -0,0 +1,165 @@
+---
+status: proposed
```

Comment:
oh you know what @mattmoor  - technically we need to update this to "implementable" - technically this should have happened before https://github.com/tektoncd/pipeline/pull/4215 went in

it's a weird part of our process - often after a lot of back and forth on the initial PR we just update it to "implementable" right from the start, which we should have probably remembered to suggest here 🤦‍♀️ 

at this point maybe you could just update the proposal with "implemented"?

Reply:
Sure, TIL.  Will send a PR shortly.

Reply:
https://github.com/tektoncd/community/pull/518